packages/core/src/render3/component.ts

```diff
@@ -146,7 +146,7 @@ export function renderComponent<T>(
     // Create element node at index 0 in data array
     elementHost(opts.host || componentDef.tag, componentDef);
     // Create directive instance with n() and store at index 1 in data array (el is 0)
-    component = directiveCreate(1, componentDef.n(), componentDef);
+    component = directive(1, componentDef.n(), componentDef);
```

Comment:
WFIW I don't like name (vs verb) as function name. I think this renaming (referring to the PR) is confusing.

Reply:
I agree but was not able to convince @mhevery

Reply:
This is the result of a 30 min+ discussion between Jeremy, Misko, and I to address renaming suggestions on https://github.com/angular/angular/pull/20855 and compromise on names. If you want to put your two cents in for what you'd rather have, we can have another summit? But I think it will be hard to get everyone to agree.

Reply:
Forget the context of this PR.

Does the following make any sense ? `pasta(Kind.Spaghetti); // eat, cook, buy, other ?`

Reply:
@mhevery It is your call

Reply:
Sorry I think we will just go with what we have for now. We can bikeshed more later.